diff --git a/dist/code.d.ts b/dist/code.d.ts
index 544e8b4c7a118cd2b928ec2fec6b9a5f4a2a4bd8..d99b0856c6dd1e7d06e26f8ed08a8904d52e37a0 100644
--- a/dist/code.d.ts
+++ b/dist/code.d.ts
@@ -1,6 +1,6 @@
 import type { GraphQLSchema } from 'graphql';
 import type { TypeScriptPluginConfig } from '@graphql-codegen/typescript';
-import type { TypeScriptResolversPluginConfig } from '@graphql-codegen/typescript-resolvers/config';
+import type { TypeScriptResolversPluginConfig } from '@graphql-codegen/typescript-resolvers';
 import type { MercuriusLoadersPluginConfig } from './mercuriusLoaders';
 declare type MidCodegenPluginsConfig = TypeScriptPluginConfig & TypeScriptResolversPluginConfig & MercuriusLoadersPluginConfig;
 export interface CodegenPluginsConfig extends MidCodegenPluginsConfig {